packages/@tinacms/sharedctx/src/edit-state-ctx.tsx

```diff
@@ -20,7 +20,7 @@ const isSSR = typeof window === 'undefined'
 
 export const isEditing = (): boolean => {
   if (!isSSR) {
-    const isEdit = window.localStorage.getItem(LOCALSTORAGEKEY)
+    const isEdit = window.localStorage && window.localStorage.getItem(LOCALSTORAGEKEY)
```

Comment:
```suggestion
```suggestion
    // localStorage may be intentionally disabled for site visitors, 
    // in that case Tina can't be used so just return false
    const isEdit = window.localStorage && window.localStorage.getItem(LOCALSTORAGEKEY)
```
    const isEdit = window.localStorage && window.localStorage.getItem(LOCALSTORAGEKEY)
```

Reply:
Oh ok, so when your editors use Tina they _will_ have access to localStorage then, correct? That seems fine, and looks like this check would do the trick 👍

Reply:
Yes, that's right ! People needing to edit via Tina will know to enable local storage API access for the site in question 😉